pytest_xmlcomp/plugin.py

```diff
@@ -118,16 +118,23 @@ def __init__(self, xpath, parent, expresult, tree):
         :type parent: :class:`_pytest.main.Session`
 
         :param expresult: the result which will be expected.
+        :param tree: the tree of the XML file
+        :param namesp: the namespaces specified in the JSON file
+        :type xpath: xpath object
+        :type parent: :class:`_pytest.main.Session'
+
         :type expresult: JSON object
 
         :param tree: the tree of the XML file
         :type tree: class lxml.ElementTree
+        :tyep namesp: dict with namespaces from JSON file
```

Comment:
* Typo: `tyep` -> `type`
* The `type` statement is just for adding type annotation. Add your descriptions into `:param:`. I would recommend to use:

```
:param namesp: the namespaces specified in the JSON file
:type namesp: :class:`dict`
```